website/docs/cloud-docs/vcs/github-app.mdx

```diff
@@ -98,12 +98,14 @@ If you disallow access to a repository that is currently connected to any Terraf
 
 ## Feature Limitations
 
-You can use the Terraform Cloud GitHub App to create workspaces and private registry modules from the Terraform Cloud UI, the API or the `tfe` Terraform provider. These features are only supported in the following releases:
+You can use the Terraform Cloud GitHub App to create workspaces and private registry modules from the Terraform Cloud UI, the API or the tfe Terraform provider. These features are only supported in the following releases:
```

Comment:
> I'm curious why you removed the code font from tfe? I think we should keep the code font or add all caps (e.g., tfe or TFE) to denote that "tfe" is a provider.
Good catch. I removed it because a code font did not make sense. But I did not think of caps at that moment. 😅 
Caps is a good idea!

Reply:
The UI works with TFE v202302-1 and later. Does it still make sense to move this to the top?

> These features are supported in any version of Terraform Cloud or in Terraform Enterprise v202303-1 and above:

Reply:
I feel like this is more clear, but please let me know if you think those last two bullet points don't match up as well in this configuration.